### DISCUSSION TITLE
Set UpdateOptions in native mongodb methods

### DISCUSSION THREAD

**Discussion by jearangoo** [mikro-orm/mikro-orm#2183] [Ideas]
**Is your feature request related to a problem? Please describe.**

hello, i am trying to use nativeUpdate in mongodb with options 
```
nativeUpdate(where: FilterQuery<T>, data: EntityData<T>, options?: UpdateOptions<T>): Promise<number>;

// UpdateOptions is this object
export interface UpdateOptions<T> {
    filters?: Dictionary<boolean | Dictionary> | string[] | boolean;
}
``` 
but the UpdateOptions are not like mongodb driver's updateMany options
https://docs.mongodb.com/manual/reference/method/db.collection.updateMany/#syntax
```
db.collection.updateMany(filter, update, options)
```
where options is like the following object
```
db.collection.updateMany(
   <filter>,
   <update>,
   {
     upsert: <boolean>,
     writeConcern: <document>,
     collation: <document>,
     arrayFilters: [ <filterdocument1>, ... ],
     hint:  <document|string>        // Available starting in MongoDB 4.2.1
   }
)
```

**Describe the solution you'd like**

the options object of the function nativeUpdate should allow to pass the correct options to the mongodb driver
i think this problem also happens to the other native methods nativeInsert, nativeDelete, aggregate
https://mikro-orm.io/docs/usage-with-mongo#native-collection-methods

**Additional context**
@mikro-orm/core: 4.5.7
@mikro-orm/mongodb: 4.5.7
nodejs: 16.4.2

**Comment by B4nan** [comment]
It's a different method, with different options - as the docs says, it is a shared interface for all drivers, you can't expect driver specific options in there. 

If you want to use the `db.collection.updateMany`, you can do so too:

```ts
// note that you need to have the EM type imported from `@mikro-orm/mongodb` to access this method
const res = await em.getCollection(EntityName).updateMany(...);
```

**Comment by jearangoo** [comment]
thank you, i couldn't find the `em.getCollection()` method in the documentation, i had to use the driver `client.db().collection()` method, but i like more `em.getCollection()`
